operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfiguration.java

```diff
@@ -390,6 +390,8 @@ public static class AccessControl {
         protected String authorizerClass = null;
         @JsonProperty("config-prefix")
         protected String configPrefix = null;
+        @JsonProperty("old-config-prefix")
+        protected String oldConfigPrefix = null;
```

Comment:
Not a fan of using `oldConfigPrefix` if there is any chance of this change hanging around, but I'm struggling to find a better name. 

One alternative that comes to mind is given this is part of  `KafkaInstanceConfiguration` (technically its in the `AccessControl` class so maybe the scope is already explicit enough?) maybe the new property should be called  `brokerPluginsConfigPrefix` to distinguish it from config for other plug-ins?

Reply:
Sounds good, I'll update to that and rebase (it's currently conflicting)